### PR TITLE
Modulo

### PR DESCRIPTION
**Pull request by MaximeJdDS**
We can now write '2 mod 2' to compute the modulo 2 of 2 ( works with the word 'modulo' too ).
I tried to follow the code style as much as possible. Hoping is enough. 

Happy coding :D

### CODE REVIEW

File: verbcalc/core/translator/symbols.py [hunk 1]
```diff
@@ -86,3 +92,8 @@ def exponentiations(self, value) -> None:
     def absolutes(self, value) -> None:
         self._absolutes = value
         self.symbol_dictionary.update({'abs': self._absolutes})
+
+    @modulo.setter
+    def logarithm(self, value) -> None:
```
Comment (ErykPiasecki07) [inline verbcalc/core/translator/symbols.py:97]: Shouldn't it be called modulo?

**Review by ErykPiasecki07 — CHANGES_REQUESTED** [review]
Hi! Great job, there's only one thing left, you named the setter 'logarithm' instead of 'modulo' (or I am just bad at math and there's some correlation I don't see ;) )

File: verbcalc/core/translator/symbols.py (hunk 1 above)
Comment (ErykPiasecki07) [reply verbcalc/core/translator/symbols.py:97]: Oh, and by the way, there's something called linked issues on GitHub pull requests. If you type in your PR's description: 'Resolves #31' (or `#` followed by any issue number), GitHub will treat both as directly related, so when the pull request gets merged, the issue will close automatically ;)

Comment (MaximeJdDS) [reply verbcalc/core/translator/symbols.py:97]: Yes ! For sure ! It a residual of my previous idea, sorry...

Have a sweet sunday. :)

**Review by ErykPiasecki07 — APPROVED** [review]

**Comment by ErykPiasecki07** [comment]
Hi @MaximeJdDS!
I forgot to create the AUTHORS.MD file at the beginning of repo's life, so if you wanna be listed for your contribution, let me know with an email of yours.

Have a nice day!

**Pull request merged**